src/routes/excel/excelRoute.ts

```diff
@@ -1,14 +1,17 @@
-import { FastifyInstance, FastifyReply, FastifyRequest } from "fastify";
-import ExcelParserService from "../../services/excelParser";
-import { Table } from "../../types/table.types";
+import { FastifyInstance } from "fastify";
+import ExcelService from "../../services/excelService";
+import { DataSource } from "../../types/excel.types";
 import { excelPostSchema } from "./schemas";
 
 const excelRoute = async (fastify: FastifyInstance) => {
 
-    fastify.post<{Body: {table: Table}}>('/parse', {schema: excelPostSchema}, async (request, reply) => {
-        const table = request.body.table;
-        const parsedTable = new ExcelParserService().parseTable(table);
-        return parsedTable;
+    fastify.post<{ Body: DataSource; }>('/add', { schema: excelPostSchema }, async (request, reply) => {
```

Comment:
I feel like add is not quite descriptive enough, try thinking about a better route name but you can also just leave it be for the time being